docs/src/content/pages/installation-next-js.mdoc

```diff
@@ -102,17 +102,9 @@ export default function Page() {
 
 ```
 
-Great! So now, we should be able to visit the `/keystatic` router in the browser.
+We're not done yet, but we should be able to visit the `/keystatic` route in the browser.
```

Comment:
```suggestion
```

(I wouldn't invite them to try using it if they need to do more things before it actually works but again, don't mind strongly)